parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -351,16 +388,14 @@ pub mod pallet {
 
 	impl<T: Config> ProcessMessage for Pallet<T> {
 		type Origin = AggregateMessageOrigin;
+		#[allow(clippy::clone_double_ref)]
 		fn process_message(
 			message: &[u8],
-			_: Self::Origin,
+			_origin: Self::Origin,
```

Comment:
This is the origin you need to use to determine the message priority. Not the origin in the encoded message.